RELEASE-NOTES.md

```diff
@@ -50,6 +50,7 @@
 * Added support for runtime version in snowpark procedures ad functions.
 * You can include previously uploaded packages in your functions, by listing them under `imports` in `snowflake.yml`
 * Added more options to `snow connection add` - now you can also specify authenticator and path to private key
+* Added `-s` flag to `snow sql` command, that shows comments. Comments, now are not shown by default
```

Comment:
Maybe `Added "--show-comments"`?

Reply:
How about `--comments` and `--no-comments`? I think typer will support this out of the box:
https://typer.tiangolo.com/tutorial/parameter-types/bool/

Reply:
Done